TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/tests/test_views.py

```diff
@@ -104,34 +110,54 @@ def test_osi_start_migration_assume_role(self):
                                     content_type='application/json')
 
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertIn('Timestamp', response.data)
+        self.assertIn('timestamp', response.data)
 
     @patch('botocore.client.BaseClient._make_api_call', new=mock_make_api_call)
     def test_osi_stop_migration(self):
         response = self.client.post('/orchestrator/osi-stop-migration', data=VALID_UPDATE_PAYLOAD,
                                     content_type='application/json')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertIn('Timestamp', response.data)
+        self.assertIn('timestamp', response.data)
 
     @mock_aws
     @patch('botocore.client.BaseClient._make_api_call', new=mock_make_api_call)
     def test_osi_stop_migration_assume_role(self):
         response = self.client.post('/orchestrator/osi-stop-migration', data=VALID_ASSUME_ROLE_UPDATE_PAYLOAD,
                                     content_type='application/json')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertIn('Timestamp', response.data)
+        self.assertIn('timestamp', response.data)
 
     @patch('botocore.client.BaseClient._make_api_call', new=mock_make_api_call)
     def test_osi_delete_migration(self):
         response = self.client.post('/orchestrator/osi-delete-migration', data=VALID_UPDATE_PAYLOAD,
                                     content_type='application/json')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertIn('Timestamp', response.data)
+        self.assertIn('timestamp', response.data)
 
     @mock_aws
     @patch('botocore.client.BaseClient._make_api_call', new=mock_make_api_call)
     def test_osi_delete_migration_assume_role(self):
         response = self.client.post('/orchestrator/osi-delete-migration', data=VALID_ASSUME_ROLE_UPDATE_PAYLOAD,
                                     content_type='application/json')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertIn('Timestamp', response.data)
+        self.assertIn('timestamp', response.data)
+
+    @patch('botocore.client.BaseClient._make_api_call', new=mock_make_api_call)
+    def test_osi_get_status_migration(self):
+        response = self.client.post('/orchestrator/osi-get-status-migration', data=VALID_UPDATE_PAYLOAD,
```

Comment:
Why is get status a 'post'?

Reply:
I did this for convenience initially and forgot to change this to a GET request with a required URL parameter. Will update this

Reply:
Change has been added now